enclave-tls/src/Makefile

```diff
@@ -77,6 +77,26 @@ $(addprefix $(Build_Libdir)/,$(attesters) $(verifiers)):
 	name=$${n#$${dir}_}; \
 	make -C $(this_dir)/$${dir}s/$$name
 
+util/toml.o: util/toml.c util/toml.h
+	$(CC) -fPIC -o $@ -c $<
+
+toml_h_uri ?= \
+	https://raw.githubusercontent.com/cktan/tomlc99/master/toml.h
+
+toml_c_uri ?= \
+	https://raw.githubusercontent.com/cktan/tomlc99/master/toml.c
+
+util/toml.c:
+	if [ ! -e "util/toml.c" ]; then \
+		wget $(toml_c_uri) -P util/; \
+	fi
+
+util/toml.h:
+	if [ ! -e "util/toml.h" ]; then \
+		wget $(toml_h_uri) -P util/; \
+		cp util/toml.h ./include/enclave-tls/; \
+	fi
```

Comment:
Why not integrate them directly?

Reply:
integrating them is simple. But the code is from the third party. How to treat this situation?

Reply:
First check its license. If it is GPL, we have to drop them and search another code implementation.